src/ray/raylet/format/node_manager.fbs

```diff
@@ -150,18 +152,20 @@ table RegisterClientReply {
   port: int;
 }
 
-table RegisterClientWithPortRequest {
+table RegisterWorkerWithPortRequest {
   // Request to register client.
   request_client_request: RegisterClientRequest;
   // Request to assign port.
   announcement_port_request: AnnounceWorkerPort;
 }
 
-table RegisterClientWithPortResponse {
-  // Response to register client.
-  request_client_request: [RegisterClientReply];
-  // Response to assign port.
-  announcement_port_request: [AnnounceWorkerPortReply];
+table RegisterWorkerWithPortReply {
+  // GCS NodeID of the local node manager.
+  raylet_id: string;
+  // Whether the announcement and job registration succeeded.
+  success: bool;
```

Comment:
These two are not needed for workers?

Reply:
raylet_id is passed in via `--raylet-id`

Reply:
Got it, I initially add it to keep compatibility with current response.

Reply:
Fixed.